tests/web_client_test.py

```diff
@@ -182,7 +182,7 @@ def testWebClientSpec(self):
                     hasJasmine = True
                 if 'Testing Finished' in line:
                     jasmineFinished = True
-                sys.stdout.write(line)
+                sys.stdout.write(line.encode('ascii', 'replace'))
```

Comment:
It's trying to internally coerce it and using the default 'ascii' codec because it is being passed a string rather than bytes, but I'm curious if we pass it utf8-encoded bytes if it will be able to display them rather than having to hide all the non-ascii.